app/views/checkout/thankyou.scala.html

```diff
@@ -1,34 +1,33 @@
 @import com.gu.i18n.Currency
 @import com.gu.memsub.promo.Promotion.AnyPromotion
-@import views.support.BillingPeriod._
+@import com.gu.memsub.subsv2.Subscription
+@import com.gu.memsub.subsv2.SubscriptionPlan.ContentSubscription
+@import model.SubscriptionOps._
 @import views.support.PlanOps._
-@import com.gu.memsub.subsv2.CatalogPlan
-@import com.gu.memsub.Product
 
-@(subscriptionName: String,
+@(  subscription: Subscription[ContentSubscription],
     guestAccountForm: Option[Form[model.GuestAccountData]] = None,
     touchpointBackendResolution: services.TouchpointBackend.Resolution,
-    plan: CatalogPlan.Paid, // we should take a subscription here
     promotion: Option[AnyPromotion] = None,
-    currency: Currency,
     startDate: String
 )(implicit request: RequestHeader)
 
-@main(s"Confirmation | Subscribe to the ${plan.title} | The Guardian", bodyClasses=List("is-wide"), touchpointBackendResolutionOpt = Some(touchpointBackendResolution), product = Some(plan.product)) {
+
+@main(s"Confirmation | Subscribe to the ${subscription.firstPlan.title} | The Guardian", bodyClasses=List("is-wide"), touchpointBackendResolutionOpt = Some(touchpointBackendResolution), product = Some(subscription.firstProduct)) {
 
 <script type="text/javascript">
     guardian.pageInfo.slug="GuardianDigiPack:Order Complete";
     guardian.pageInfo.productData = {
-        numberOfMonths: @{plan.charges.billingPeriod.monthsInPeriod},
+        numberOfMonths: @{subscription.firstPlan.charges.billingPeriod.monthsInPeriod},
```

Comment:
I think it was a requirement for Omniture. Google Analytics does not use it, so it can go. Also delete line 22 in assets/scripts/modules/checkout/eventTracking.js.